classes/class-ledyer-om-credentials.php

```diff
@@ -22,9 +22,12 @@ class Credentials {
 	 * Credentials constructor.
 	 */
 	public function set_settings() {
-		// First try and get credentials from Ledyer checkout
+		// First try and get credentials from Ledyer checkout.
 		self::$settings = get_option( 'woocommerce_lco_settings' );
-		// If that is not found, try and get from Ledyer Payments (to be developed)
+		// If that is not found, try and get from Ledyer Payments.
+		if ( empty( self::$settings ) ) {
+			self::$settings = get_option( 'woocommerce_ledyer_payments_settings' );
+		}
```

Comment:
Since we get the credentials from here, would this not make it so that if you have both, or previously had checkout, the credentials from checkout would always be used even for Ledyer Payments? Since this is just for order management, should we maybe refactor this to load the settings based on the payment method from the order instead?